src/ethereum_spec_tools/evm_tools/utils.py

```diff
@@ -26,6 +27,9 @@
 W = TypeVar("W", Uint, U64, U256)
 
 EXCEPTION_MAPS = {
+    "BPO1": {
+        "fork_blocks": [("osaka", 0)],
+    },
```

Comment:
We need this to fill all the tests!
```suggestion
    "BPO1": {
        "fork_blocks": [("osaka", 0)],
    },
    "BPO2": {
        "fork_blocks": [("osaka", 0)],
    },
    "BPO3": {
        "fork_blocks": [("osaka", 0)],
    },
    "BPO4": {
        "fork_blocks": [("osaka", 0)],
    },
```